homeassistant/components/overkiz/water_heater_entities/atlantic_domestic_hot_water_production_mlb_component.py

```diff
@@ -140,6 +145,7 @@ async def async_set_operation_mode(self, operation_mode: str) -> None:
         elif operation_mode in (
             OverkizCommandParam.MANUAL,
             OverkizCommandParam.MANUAL_ECO_INACTIVE,
```

Comment:
This service handler will also need to be updated to only accept the allowed operation mode states as input.

Reply:
I did not understand you, or `async_set_operation_mode` supports `STATE_ELECTRIC` at line 148.

Reply:
Yes, but these other modes that are not part of the allowed states need to be removed too.

Reply:
Sorry, got your point. Done.